source/jormungandr/jormungandr/interfaces/v1/Ptobjects.py

```diff
@@ -97,5 +98,8 @@ def get(self, region=None, lon=None, lat=None):
             abort(400, message="Search word absent")
         response = i_manager.dispatch(args, "pt_objects",
                                       instance_name=self.region)
+        response = marshal(response, pt_objects)
+        if i_manager.instances[self.region].bss_provider and args["bss_stands"]:
```

Comment:
pt_objects don't return POI, it only provide transport objects (like it's name said).
If you want stands availability in the autocompletion it's the places api that need to be modified.
If it's in /pois it is in Uri.py, but we probably need to do something a little smarter here since it's a "meta" API.

I am not sure we want to query an external service for stands availability in places, it will have severe impact on performance, and if we do, it will probably be better so set the default value of ```bss_stands``` to False, this way we don't have to pay the cost of something if we don't want to use it.